### PR TITLE
feat: enable send message option during decline PAs AB#29108

### PR DESCRIPTION
**Pull request by ashishkumarbasak-accenture**
## Describe your changes

Enable the functionality to decline PAs with message. Also the message should should visible at PAs activity overview.

Task:  [29108](https://dev.azure.com/redcrossnl/121%20Platform/_workitems/edit/28988)

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] I have added tests wherever relevant
- [x] I have made sure that all automated checks pass before requesting a review
- [x] I do not need any deviation from our PR guidelines

### CODE REVIEW

**Review by aberonni — APPROVED** [review]
I didn't test the functionality but the code looks good. Could you double-check with Sanne that it works as expected before merging? Then we can release it.

**Comment by ashishkumarbasak-accenture** [comment]
> I didn't test the functionality but the code looks good. Could you double-check with Sanne that it works as expected before merging? Then we can release it.

I just asked Sanne, waiting for her reply!

**Pull request merged**
